WordPress/src/main/java/org/wordpress/android/ui/posts/editor/EditorMedia.kt

```diff
@@ -240,16 +233,10 @@ class EditorMedia(
         mimeType: String?,
         startingState: MediaUploadState
     ): MediaModel? {
-        val media = FluxCUtils.mediaModelFromLocalUri(
-                activity,
-                uri,
-                mimeType,
-                mediaStore,
-                site.id
-        ) ?: return null
-        if (org.wordpress.android.fluxc.utils.MediaUtils.isVideoMimeType(media.mimeType)) {
-            val path = MediaUtils.getRealPathFromURI(activity, uri)
-            media.thumbnailUrl = getVideoThumbnail(path)
+        val media = fluxCUtilsWrapper.mediaModelFromLocalUri(uri, mimeType, mediaStore, site.id) ?: return null
```

Comment:
We might want to consider using a single return statement here. Perhaps something like this

```
return fluxCUtilsWrapper.mediaModelFromLocalUri(uri, mimeType, mediaStore, site.id)?.let{ media ->
            if (mediaUtilsWrapper.isVideoMimeType(media.mimeType)) {
                val path = mediaUtilsWrapper.getRealPathFromURI(uri)
                media.thumbnailUrl = EditorMediaUtils.getVideoThumbnail(activity, path)
            }

            media.setUploadState(startingState)
            editorMediaListener.editorMediaPostData().let {
                if (!it.isLocalDraft) {
                    media.postId = it.remotePostId
                }
            }
            media
        }
```

Wdyt?

Reply:
Fixed in 5b70d2423f951be95ce43313e7d6b40c1c53c8ac.